src/utils/convert.spec.ts

```diff
@@ -231,10 +232,10 @@ describe('it Remove / Add commission from amount', () => {
   }) => {
     const amountBN = new BigNumberJs(amount);
     const rateBN = new BigNumberJs((substract ? 100 - rate : 100 + rate) / 100);
-    return amountBN.times(rateBN).toNumber();
+    return bigify(trimBN(amountBN.times(rateBN).toString()));
```

Comment:
Can this be a BigNumber or hex-string rather than a decimal string?

Reply:
No, this is intentional since `trimBN` takes a decimal string. Also it's a test so not very important imo.